crates/pop-common/src/git.rs

```diff
@@ -491,6 +499,23 @@ mod tests {
 		);
 	}
 
+	#[test]
+	fn parse_latest_tag_works() {
+		let mut tags = vec![];
+		assert_eq!(Git::parse_latest_tag(tags), None);
+		tags = vec![
+			"polkadot-v1.10.0",
+			"polkadot-v1.11.0",
+			"polkadot-v1.12.0",
```

Comment:
You're right; we need to start supporting these new tag labels. However, I'm not entirely sure what the best approach would be. The main goal of the `parse_latest_tag` function is to identify the latest version to use as a default when the user doesn't specify one.

One challenge is figuring out how to compare the "stable" tag with the other versions. For example, we have tags like `polkadot-stable2407`, and `polkadot-v1.15.0`. How should we determine which one is the latest?

Reply:
You are right. Based on my understanding from this discussion: https://forum.polkadot.network/t/stabilizing-polkadot/7175/26, the new releases will follow the format `polkadot-stable2407` (changing every 3 months) and `polkadot-stable2407-1 `for fixes. So added support for this with more test cases

Reply:
Since the goal of `parse_latest_tag` is to determine the latest version to use by default when the user doesn't specify one, we can ignore pre-release labels like `-rc`. I've included this adjustment as well.